MapboxDirections/MBAnnotation.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  MBAnnotationType.swift
+//  MapboxDirections
+//
+//  Created by Bobby Sudekum on 3/20/17.
+//  Copyright © 2017 Mapbox. All rights reserved.
+//
+
+import Foundation
+
+@objc(AnnotationType)
+public enum AnnotationType: Int, CustomStringConvertible {
+
+    case congestion
+    
+    case distance
+    
+    case duration
```

Comment:
"Durations" can indicate lots of things. Let's call this `expectedTravelTime`, for consistency with the `expectedTravelTime` property of RouteLeg and RouteStep.